src/pybind/mgr/dashboard/frontend/cypress/e2e/ui/language.e2e-spec.ts

```diff
@@ -14,6 +14,6 @@ describe('Shared pages', () => {
 
   it('should check all available languages', () => {
     language.getLanguageBtn().click();
-    language.getAllLanguages().should('have.length', 1).should('contain.text', 'English');
+    language.getAllLanguages().should('have.length', 13).should('contain.text', 'English');
```

Comment:
TBH, I personally don't like this type of tests, as they validate input configuration rather than behaviour,  so I'd rather remove it (or replace it with a general expression, like ">=1"). But given you fixed it your way, let's keep it! 👍🏼